src/vs/workbench/contrib/notebook/browser/viewParts/notebookEditorStickyScroll.ts

```diff
@@ -54,26 +56,46 @@ export class ToggleNotebookStickyScroll extends Action2 {
 export class NotebookStickyLine extends Disposable {
 	constructor(
 		public readonly element: HTMLElement,
+		public readonly foldingIcon: StickyFoldingIcon,
+		public readonly header: HTMLElement,
 		public readonly entry: OutlineEntry,
 		public readonly notebookEditor: INotebookEditor,
 	) {
 		super();
-		this._register(DOM.addDisposableListener(this.element, DOM.EventType.CLICK, () => {
+		// click the header to focus the cell
```

Comment:
We might want to listen to `EventType.CLICK` and `TouchEventType.Tap` to ensure it's also working with touch screen